docs/convolution/performance.inc.rst

```diff
@@ -8,5 +8,6 @@
 Performance Tips
 ================
 
-Here we provide some tips and tricks for how to optimize performance of code
-using `astropy.convolution`.
+The :func:`~astropy.convolution.convolve` function is best suited to small
+kernels, and can become very slow for larger kernels. In this case, consider
```

Comment:
The :func:``~astropy.convolution.convolve_fft`` has a substantially larger memory footprint than the :func:``~astropy.convolution.convolve``. If memory consumption is close to the resource limit, using ``~astropy.convolution.convolve`` maybe significantly more performant.

Reply:
Ok, I mention the higher memory usage now.